lambdas/build-proven-user-identity-details/src/main/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/BuildProvenUserIdentityDetailsHandler.java

```diff
@@ -85,6 +85,7 @@ public APIGatewayProxyResponseEvent handleRequest(
             configService.setFeatureSet(RequestHelper.getFeatureSet(input));
             String ipvSessionId = RequestHelper.getIpvSessionId(input);
             IpvSessionItem ipvSessionItem = ipvSessionService.getIpvSession(ipvSessionId);
+            LogHelper.attachIpvSessionIdToLogs(ipvSessionId);
```

Comment:
I believe `RequestHelper.getIpvSessionId` should actually be attaching session id to the logs itself (which I don't really like as it's a side effect 😕) so I imagine the only cases we're actually missing session id is where we're not using the RequestHelper method for some reason - could you check this?